### DISCUSSION TITLE
Generating topology for Dextran Sulfate polymer

### DISCUSSION THREAD

**Discussion by ajhaver4** [marrink-lab/polyply_1.0#302] [Q&A] [answered]
Hi, 
I am looking to simulate a protein-polymer system which has Dextran sulfate of the polymer. Since currently the martini3 library only has the dextran molecule, how can I generate my own topology and .itp files for dextran sulfate using polyply?

There was a related post where a user was able to generate a coarse grained model of glucose with sulfate. However I'm not clear on how they did that.

**Comment by fgrunewald** [comment, marked as answer]
Hi @ajhaver4, 

Currently we don't have parameters for dextran sulfate available. We are planning to create some in the near future, but I cannot promise when they will be done. I'm also not sure what happened in the other case. 

Cheers,
Fabian
